css-box-model/styles.css

```diff
@@ -0,0 +1,7 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin: 16px 12px;
```

Comment:
The instructions don't say anything about top or bottom margin:

---

![image](https://user-images.githubusercontent.com/16143771/167922230-01d5871d-8f0f-489d-b521-e04939a0c234.png)